examples/benchmarks/bubblebench/baseline/package.json

```diff
@@ -11,13 +11,26 @@
 	},
 	"license": "MIT",
 	"author": "Microsoft and contributors",
+	"type": "module",
+	"exports": {
+		".": {
+			"import": {
+				"types": "./lib/index.d.ts",
+				"default": "./lib/index.js"
+			},
+			"require": {
+				"types": "./dist/index.d.ts",
+				"default": "./dist/index.js"
+			}
+		}
+	},
 	"main": "dist/index.js",
-	"module": "lib/index.js",
 	"types": "dist/index.d.ts",
 	"scripts": {
```

Comment:
Nit: As Jason pointed out on one of my PRs today, we can probably remove the entrypoints ('exports', 'main', and 'types') since this is an app.

Reply:
Although we don't consume it, I think this one is setup to be a library.
It has exports for Bubblebench and BubblebenchInstantiationFactory.